frontend/src/components/form-container/FormContainer.js

```diff
@@ -28,7 +28,7 @@ const FormContainer = ({ navigate }) => {
   });
 
   useEffect(() => {
-    const animationDuration = 3000;
+    const animationDuration = 1;
```

Comment:
dont forget to reset this to 3000 prior to push :)

Reply:
ty! done